src/translations/en.json

```diff
@@ -524,5 +544,10 @@
     "heading": "World Championship IV",
     "paragraph": "The Cuttle World Championship IV is today, Saturday March 8th at 12pm EST! Watch the top 8 players from the past YEAR of competitive play battle it out for the title of Cuttle World Champion and their pick from the 8 incredible prize decks donated by player SUBMARINO! Watch it live at ",
     "twitchLink": "twitch.tv/cuttle_cards"
+  },
+  "other": {
+    "addGameToMatch": "Could not add game to match",
+    "dontWipeDbOnProd": "Error: This action is only permitted in development or staging",
```

Comment:
Note to self: don't wipe db on prod

Reply:
Let's put `"dontWipeDbOnProd"` in the top-level `"error"` block at the very bottom. This one is the weirdest because we really shouldn't be hitting it outside of testing (thank goodness haha) so it doesn't fit super neatly into the normal space of errors. It's the one that by itself might warrant creating an "other" section live you have here, but the other two have more well-defined spots they belong in imo so instead we'll throw this in `"error"` down at the bottom